salt/backups/templates/restore_mongodb.sh

```diff
@@ -4,14 +4,16 @@ set -e
 {% set backupdir = '/backups/{}'.format(settings.get('directory', 'mongodb')) %}
 {% set cachedir = '/backups/.cache/{}'.format(settings.get('directory', 'mongodb')) %}
 mkdir -p {{ backupdir }}
+mkdir -p {{ cachedir }}
 
 PASSPHRASE={{ settings.duplicity_passphrase }} /usr/bin/duplicity restore\
-          --s3-use-server-side-encryption s3+http://odl-operations-backups/{{ settings.get('directory', 'mongodb') }}/ \
-          --archive-dir {{ cachedir }}
+          --s3-use-server-side-encryption s3+http://odl-operations-backups/{{ settings.get('directory', 'mongodb') }} \
+          --archive-dir {{ cachedir }} \
           --force --tempdir /backups/tmp/ {{ backupdir }}
 
 /usr/bin/mongorestore --host {{ settings.host }} \
                       --port {{ settings.get('port', 27017) }} \
                       --password {{ settings.password }} --username admin \
                       --authenticationDatabase admin \
-                      {{ backupdir }}
+                      --db {{ settings.database }} \
+                      --drop {{ backupdir }}/{{ settings.database }}
```

Comment:
To support mapping between source/destination databases for restore we will need to iterate over a mapping for the `mongorestore` piece, but this is good as a first step to bring everything in line. We can do that as a second PR or second commit.